api/lib/index.js

```diff
@@ -34,6 +35,20 @@ const lib = {
 
     return false;
   },
+  isRangeOfNumbers: str => {
+    if (typeof str !== 'string' && Number.isNaN(str)) {
```

Comment:
why not do a `typeof str === 'number' && Number.isNaN(str)` instead?

Reply:
I think you need to do a parseInt before checking for isNaN, it not that useful otherwise. 
![image](https://user-images.githubusercontent.com/2982784/70378062-03405280-18d1-11ea-889a-06b8331fa26f.png)

Reply:
I need advice. The  type can either be a string or number. Is it worth even checking in that  case?

Reply:
you might be able to remove this condition, the string regex will catch strings that don't match. The only input for this function is a string from what I can see.